features/support/page_objects/confirmation_page_refund.rb

```diff
@@ -0,0 +1,30 @@
+class ConfirmationPageRefund < BasePage
+  set_url '/confirmation/refund'
+
```

Comment:
I look at the code and only place I can see you used this page is to check header. So I see no reason why you have the rest of the page object copied from confirmation page too?